File: TorahWeb/ios/LiquidGlass/ios/TorahWebLiquidGlassViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE (TorahWebLiquidGlassViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(variant, NSString)
RCT_EXPORT_VIEW_PROPERTY(glassTintColor, UIColor)
RCT_EXPORT_VIEW_PROPERTY(cornerRadius, CGFloat)
RCT_EXPORT_VIEW_PROPERTY(isInteractive, BOOL)

@end

File: TorahWeb/ios/LiquidGlass/ios/TorahWebLiquidGlassViewManager.swift
import Foundation
import React

@objc(TorahWebLiquidGlassViewManager)
class TorahWebLiquidGlassViewManager: RCTViewManager {
  override func view() -> UIView! {
    return TorahWebLiquidGlassView()
  }

  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}